README.md

```diff
@@ -5,19 +5,20 @@ Provides Azure Authentication Middleware for a Laravel App.
 ## Normal Installation
 
 1. `composer require rootinc/laravel-azure-middleware`
-2. In our routes folder (most likely `web.php`), add
+2. Copy the file `/vendor/rootinc/laravel-azure-middleware/config/azure.php` to your application's `/config` directory
```

Comment:
Could we have the instructions publish the package config instead of having to copy/paste from the vendor directory?

https://laravel.com/docs/5.8/packages#configuration

Reply:
I have added this in, as seen in PR #16, I can split the PR if needed.